config/form_profile_mappings/10-10EZR.yml

```diff
@@ -16,4 +16,6 @@ spouseDateOfBirth: [ezr_data, spouseDateOfBirth]
 dateOfMarriage: [ezr_data, dateOfMarriage]
 cohabitedLastYear: [ezr_data, cohabitedLastYear]
 maritalStatus: [ezr_data, maritalStatus]
-# dependents: [ezr_data, dependents]
+dependents: [ezr_data, dependents]
+nonPrefill:
```

Comment:
I added this `nonPrefill` key for the financial info, since we aren't using it to actually prefill the form. Instead, it's to show users what financial info they entered previously. I saw it laid out this way in other form profiles (see https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/form_profile_mappings/686C-674-V2.yml for an example).